gnd/src/main/java/com/google/android/gnd/ui/editrecord/EditRecordViewModel.java

```diff
@@ -62,7 +62,7 @@
   private final SingleLiveEvent<Void> showUnsavedChangesDialogEvents;
   private final SingleLiveEvent<Void> showErrorDialogEvents;
   private final Resources resources;
-  private final ObservableMap<String, Value> values = new ObservableArrayMap<>();
+  private final ObservableMap<String, Response> values = new ObservableArrayMap<>();
```

Comment:
Should this be called `responses`?

It might be simpler than maintaining the distinction that value==response in this ViewModel. I might also be missing something. 

The corresponding getters would also need to be changed.

Reply:
These were overlooked due to search & replace fatigue, thanks for catching them!